src/tests/test_error_recovery.py

```
@@ -0,0 +1,353 @@
+    # Create test data
+    scenario = MockDataFactory.create_scenario("standard", claim_count=2)
+    claims = scenario["claims"]
+    evidence_map = scenario["evidence_map"]
+    verdicts = scenario["verdicts"]
+    
+    # Configure mock to simulate a timeout for the second evidence gathering
+    call_count = 0
+    def mock_runner_side_effect(*args, **kwargs):
+        nonlocal call_count
+        agent = args[0]
+        
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)